thorlcr/activities/indexwrite/thindexwriteslave.cpp

```diff
@@ -537,7 +528,8 @@ class IndexWriteSlaveActivity : public ProcessSlaveActivity, public ILookAheadSt
     {
         if (processed & THORDATALINK_STARTED)
         {
-            doStopInput();
+            if (!inputStopped) // probably already stopped in process()
```

Comment:
there really shouldn't be. But I kept it conditional as before to minimize this changes impact.